CompositeUi/src/scss/components/_grid.scss

```diff
@@ -6,35 +6,22 @@
 // For the full copyright and license information, please view the LICENSE
 // file that was distributed with this source code.
 
-@import '../base/grid';
-@import '../variables/grid';
+$grid-type: 'fluid';
```

Comment:
Import `../variables/grid` here, remove already declared variables and include grid__row('row')